### PR TITLE
Added callout for handbook

### LINKED ISSUE #2298: [Handbook] no button or part in layer5.io/community which redirect to layer5.io/community/handbook
**Current Behavior**no button or part in layer5.io/community which redirect to layer5.io/community/handbook , we always have to type the url to redirect to the handbook page

**Desired Behavior**

 -links to this handbook from the /community/newcomers and /community/meshmates pages?
 -a new call-to-action in the shape a small rectangular widget
 -a new, horizontal shaped call-to-action for the handbook
---
**Screenshots / Mockups**

**Alternatives**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by YashKamboj**
Signed-off-by: YashKamboj <75125203+YashKamboj@users.noreply.github.com>

**Description**Added callout for handbook on community page, newcomers page , and a small on learn course page

This PR fixes #2298 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit efc58db46da8fff211d8f4b8e11f68fc1c86c43d at: https://6193e05b7d352433cd47434c--layer5ng.netlify.app

**Comment by warunicorn19** [comment]
Let's not put more than 1 callout on one page. at max. 2

**Review by leecalcote — CHANGES_REQUESTED** [review]
@YashKamboj will you refactor this PR so that only intentional changes show in the diff? There are too many errant changes in this PR, making it difficult to review and approve.

**Comment by YashKamboj** [comment]
@leecalcote Yeah , removed the errant changes, they were just because of the prettier extension in my vscode.

**Comment by l5io** [comment]
🚀 Preview for commit 4a05844f01118f9d33772b2a27d94d1fdca08787 at: https://6194aaa5dc3f86bed921a727--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c065d65a827e5244a6d3dfca22097f648f57447d at: https://61a354e3360b458c9c6be1b1--layer5ng.netlify.app

**Comment by YashKamboj** [comment]
@warunicorn19 remind??

**Comment by warunicorn19** [comment]
@debo19 @Nikhil-Ladha your inputs here would be great.

File: src/sections/Community/index.js [hunk 1]
```diff
@@ -122,8 +121,9 @@ const CommunityPage = () => {
         </Container>
       </div>
       <JoinCommunity image={Lee_workshop} className="newcomers-join" />
-      <DiscussCallout />
-
+      <div className="callout">
+        <DiscussCallout />
```
[the remark this answers is not in the document]
Comment (YashKamboj) [reply src/sections/Community/index.js:125]: On which should it be then?? @debo19

File: src/sections/Learn-Layer5/Course-Overview/index.js [hunk 2]
```diff
@@ -119,6 +121,7 @@ const CourseOverview = ({ course, chapters, serviceMeshesList }) => {
             </div> */}
             <DiscussCallout />
             <SubscribeLearnPath />
+            <HandbookCallout image={handbookcallourbng} />
```
[the remark this answers is not in the document]
Comment (YashKamboj) [reply src/sections/Learn-Layer5/Course-Overview/index.js:124]: About the Layer5 banner below which shows up , we can make a new image cropping the banner @debo19

[the remark this answers is not in the document]
Comment (YashKamboj) [reply src/sections/Learn-Layer5/Course-Overview/index.js:124]: > 

The idea could be great  @debo19

File: src/sections/Community/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (debo19) [reply src/sections/Community/index.js:125]: Ideally, https://layer5.io/community/newcomers should work. Though I don't see a space here as well to fit a small call-out 🤔

[the remark this answers is not in the document]
Comment (YashKamboj) [reply src/sections/Community/index.js:125]: @debo19 we can make  a banner on the newcomers page

**Pull request closed without merging**